tag_browser_plugin/tag_browser_data.proto

```
@@ -0,0 +1,78 @@
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+syntax = "proto3";
+
+package umh.events;
```

Buf (1.47.2)

Files with package "umh.events" must be within a directory "umh/events" relative to root but were in directory "tag_browser_plugin".

(PACKAGE_DIRECTORY_MATCH)